airflow/utils/types.py

```diff
@@ -99,3 +99,18 @@ class EdgeInfoType(TypedDict):
     """Extra metadata that the DAG can store about an edge, usually generated from an EdgeModifier."""
 
     label: str | None
+
+
+class DagRunTriggeredByType(str, enum.Enum):
```

Comment:
```suggestion
class DagRunTriggeredByType(enum.Enum):
```

Existing enums subclass from str due to compatibility; we don’t need that here.

Reply:
If we remove the `str` then the db value will be in the upper case format like: `UI`, `TIMETABLE` not `ui`, `timetable`.
Do you think that this is OK, @uranusjr ?

Reply:
That’s fine, nobody should care about those values directly anyway.